js-webapp-async/README.md

```diff
@@ -25,31 +29,29 @@ Below is the code for the webapp. Notice the creation of the Worker object, the
 The webpage that uses the worker will look like:
 ```html
 <!doctype html>
-<!-- this HTML page is stored as webassembly/example-web-worker.html -->
 <html lang="en">
   <head>
     <title>Example web worker</title>
     <script>
-      // this JavaScript snippet is later referred to as <<worker-consumer>>
       const worker = new Worker('worker.js');
-      // this JavaScript snippet is appended to <<worker-consumer>>
       worker.postMessage({
         type: 'CALCULATE',
         payload: { epsilon: 0.001, guess: -20 }

```

Comment:
```suggestion
        payload: { epsilon: 0.001, guess: -4 }

```

Reply:
Watch out this also requires changes in source conde and wasm file, best to do in another PR

Reply:
I agree with Stefan and rather do this in one go.